client/shipping-label/views/steps/packages/index.js

```diff
@@ -3,16 +3,40 @@ import FoldableCard from 'components/foldable-card';
 import { translate as __ } from 'lib/mixins/i18n';
 import OrderPackages from './list';
 import ExpandButton from 'shipping-label/views/expand-button';
+import GridIcon from 'components/gridicon';
+
+const Header = ( { isValid } ) => (
+	<div>
+		<GridIcon icon={ isValid ? 'checkmark' : 'notice-outline' } />
+		{ __( 'Packages' ) }
+	</div>
+);
+
+Header.propTypes = {
+	isValid: PropTypes.bool.isRequired,
+};
+
+const Summary = ( { messageId } ) => {
+	switch ( messageId ) {
+		case 'weight':
+			return <div style={ { color: 'orange' } }>{ __( 'Weight not entered' ) }</div>;
+		default:
+			return <div>{ __( 'Bike Box: 47lbs' ) }</div>;
+	};
+};
 
 const PackagesStep = ( { values, storeOptions, labelActions, errors } ) => {
+	const isValid = true;
+	const summaryId = isValid ? 'default' : 'weight';
 	return (
 		<FoldableCard
-			header={ __( 'Packages' ) }
-			summary={ __( '' ) }
+			header={ <Header isValid={ isValid } /> }
+			summary={ <Summary messageId={ summaryId } /> }
```

Comment:
Just a heads-up: If you check #520 , I've refactored this part to make it consistent between all the steps. Instead of `FoldableCard`, use [`StepContainer`](https://github.com/Automattic/woocommerce-connect-client/pull/520/files#diff-af9c738430ebeba70167e438b0a5287eR15), it has all the Summary and Gridicons logic built-in.